rbac/providers/common/inbound_filters.py

```diff
@@ -68,17 +68,10 @@ def datetime_to_seconds(value):
 def inbound_value_filter(value):
     """Cleans up data values
     1. Unwraps LDAP attributes
-    2. Removes empty arrays
-    3. Unwraps single value arrays
-    4. Converts datetime.datetime to seconds
+    2. Converts datetime.datetime to seconds
     """
     if hasattr(value, "value"):
         value = value.value
```

Comment:
Here , we don't return,  we just set value. Below , in the elif, we return. We should be consistent (either we return in both places, or set in both places). 
In both places we should assign return value , and return that return value at the end. 
Also, the return value variable should be named differently than the input value.

We should do this: 

retValue = value

if (...)
  retValue = inbound_value.value
elif (...)
  retValue = datetime_to_seconds(inbout_value)

return retValue